client/modules/IDE/components/KeyboardShortcutModal.jsx

```diff
@@ -74,6 +74,12 @@ function KeyboardShortcutModal() {
         </span>
         <span>Turn off Accessible Output</span>
       </li>
+      <li className="keyboard-shortcut-item">
+        <span className="keyboard-shortcut__command">
+          {metaKeyName} + {'\u21E7'} + 3
```

Comment:
Also add the shortcut comand in the navbar inside sketch menu with add File option

Reply:
@shakti97 done